annotator-core/src/main/java/edu/ucr/cs/riple/core/cache/BaseCache.java

```diff
@@ -51,19 +52,19 @@ public BaseCache(S store) {
 
   @Override
   public boolean isUnknown(Fix fix) {
-    return !this.store.containsKey(fix.toLocation());
+    return !this.store.containsKey(fix.toLocations());
```

Comment:
High-level question: why not use `Fix` as the key type for the maps, instead of `Set<Location>`?

Reply:
@msridhar That was a very good catch. It should be "Fix" with the new changes on Fix declaration. In previous design "Fix" contained some other properties except Set of locations. This is much better now. d4f085445491ec1db105d8296b05fe964da88797